config/HH_bbWW/Run3_2022/samples.yaml

```diff
@@ -553,55 +555,55 @@ VBFHHto2B2V_CV_1_C2V_0_C3_1:
 VBFHHto2B2V_CV_1_C2V_1_C3_1:
   sampleType: HHnonRes
   generator: madgraph
-  #crossSection: To_be_added  
+  #crossSection: To_be_added
 VBFHHto2B2V_CV-1p74_C2V-1p37_C3-14p4:
   sampleType: HHnonRes
   generator: madgraph
-  #crossSection: To_be_added  
+  #crossSection: To_be_added
 VBFHHto2B2V_CV-m0p012_C2V-0p030_C3-10p2:
   sampleType: HHnonRes
   generator: madgraph
-  #crossSection: To_be_added  
+  #crossSection: To_be_added
 VBFHHto2B2V_CV-m0p758_C2V-1p44_C3-m19p3:
   sampleType: HHnonRes
   generator: madgraph
-  #crossSection: To_be_added  
+  #crossSection: To_be_added
 VBFHHto2B2V_CV-m0p962_C2V-0p959_C3-m1p43:
   sampleType: HHnonRes
   generator: madgraph
-  #crossSection: To_be_added  
+  #crossSection: To_be_added
 VBFHHto2B2V_CV-m1p21_C2V-1p94_C3-m0p94:
   sampleType: HHnonRes
   generator: madgraph
-  #crossSection: To_be_added  
+  #crossSection: To_be_added
 VBFHHto2B2V_CV-m1p60_C2V-2p72_C3-m1p36:
   sampleType: HHnonRes
   generator: madgraph
-  #crossSection: To_be_added  
+  #crossSection: To_be_added
 VBFHHto2B2V_CV-m2p12_C2V-3p87_C3-m5p96:
   sampleType: HHnonRes
   generator: madgraph
-  #crossSection: To_be_added  
+  #crossSection: To_be_added
 VBFHHto2B2Vto2L2Nu_CV_1_C2V_0_C3_1:
   sampleType: HHnonRes
   generator: madgraph
-  #crossSection: To_be_added  
+  #crossSection: To_be_added
 VBFHHto2B2Vto2L2Nu_CV_1_C2V_1_C3_1:
   sampleType: HHnonRes
   generator: madgraph
-  #crossSection: To_be_added  
+  #crossSection: To_be_added
 VBFHHto2B2Vto2L2Nu_CV-1p74_C2V-1p37_C3-14p4:
   sampleType: HHnonRes
   generator: madgraph
-  #crossSection: To_be_added  
+  #crossSection: To_be_added
 VBFHHto2B2Vto2L2Nu_CV-m0p012_C2V-0p030_C3-10p2:
   sampleType: HHnonRes
   generator: madgraph
-  #crossSection: To_be_added  
+  #crossSection: To_be_added
 VBFHHto2B2Vto2L2Nu_CV-m0p758_C2V-1p44_C3-m19p3:
   sampleType: HHnonRes
   generator: madgraph
-  #crossSection: To_be_added  
+  #crossSection: To_be_added
```

Comment:
AnaTuple production will not run without crossSection value. It is expected to be fixed in #63.